.github/workflows/test_unit.yml

```diff
@@ -26,21 +26,6 @@ jobs:
       matrix: ${{ fromJson(needs.resolve-modules.outputs.matrix) }}
       fail-fast: false
     runs-on: ubuntu-latest
-    services:
```

Comment:
Don't we need this to test that we successfully write to the database?

Reply:
I think let's discuss this offline as it's a bigger discussion but every destination plugin will have different requirements for databases and so on (sources dont need any db). so I think we can't automate much here but rather just have one workflow per destination plugin (and I dont think we will have more than 10 so shouldn't be an issue or a lot to automate) even if we will have to copy paste a bit

Reply:
Maybe keep the tests at least until we have the discussion, otherwise we won't have coverage for the sync command